ush/python/pygw/src/pygw/jinja.py

```diff
@@ -1,44 +1,191 @@
 import io
-import os
 import sys
 import jinja2
+from markupsafe import Markup
```

Comment:
never heard of this, is it WCOSS2 compliant?

Reply:
unfortunately the `mumbojumbo` library is probably never going to get approved for WCOSS2 :-(

Reply:
great name though 😜

Reply:
> It is already on WCOSS2. I think it is part of standard python library (not 100% sure).
> 
> ```
> 7088 clogin02 ~ ❯❯❯ module list
> 
> Currently Loaded Modules:
>   1) craype-x86-rome     (H)   3) craype-network-ofi (H)   5) craype/2.7.17        7) intel/19.1.3.304
>   2) libfabric/1.11.0.0. (H)   4) envvar/1.0               6) PrgEnv-intel/8.3.3   8) python/3.8.6
> 
>   Where:
>    H:  Hidden Module
> 
> 
> 
> 7089 clogin02 ~ ❯❯❯ python -c 'from markupsafe import Markup'
> 7090 clogin02 ~ ❯❯❯ python -c 'import mumbojumbo'
> Traceback (most recent call last):
>   File "<string>", line 1, in <module>
> ModuleNotFoundError: No module named 'mumbojumbo'
> ```

It is part of `miniconda`, not the `/usr/bin/python3` -- at least not on RDHPCS Hera.